### ISSUE TITLE
Support access tokens and authorization codes

### ISSUE THREAD

**Issue by gurisko** [gurisko/cordova-plugin-accountkit#15]
[More info](https://developers.facebook.com/docs/accountkit/accesstokens)

**Comment by 1N50MN14** [comment]
+1 :)

**Comment by 1N50MN14** [comment]
@gurisko Just wondering if you had the chance to look into the iOS part of this.. thank you ;)

**Comment by gurisko** [comment]
Hey @1N50MN14. I will be able to look at this only after February 8.

**Comment by 1N50MN14** [comment]
Cool thanks for the update @gurisko

**Comment by 1N50MN14** [comment]
Hi @gurisko , don't mean to nag, just wondering if you got the chance to look into this one - thank you ;)

**Comment by gurisko** [comment]
It's been a long time @1N50MN14 . Are you still available to test this feature?

**Comment by gurisko** [comment]
If anyone feels like it, pull request #31 is open for reviews. If there won't be any comments in around 12 hrs, it will be merged and released under 1.3.0.

**Comment by 1N50MN14** [comment]
hey @gurisko I thought you disappeared! ;) I paid $ to get the iOS part done few months ago and its tested maybe you want to take a look before merging your PR see if there's special in there?

**Comment by gurisko** [comment]
@1N50MN14 I was too busy and I stopped to use this plugin so there was a long break. Now I am trying to finish the planned features. I would definitely like to have a look if you don't mind. And thank you for testing. Hopefully we will make it better and you can get back to our public releases :)

**Comment by 1N50MN14** [comment]
@gurisko Awesome! Yeah it would be great to get back to a public release I had a concern about the long term maintainability of the plugin. The invite link should be in your inbox, couple of notes: 

1) I enforced useAccessToken in the iOS code; there's a minor issue where the arg isn't interpreted correctly and since my iOS skills are non-existent I simply enforced the use of access token.

2)  A `themeColor` option is added to both iOS and Android.

For Android I faced an issue where the AccountKit activity window by default uses FB's blue color on the status bar which looks kind of weird, what I did is replace the activity tag in AndroidManifest.xml with: `<activity android:name="com.facebook.accountkit.ui.AccountKitActivity" android:theme="@style/AccountKitTheme" tools:replace="android:theme">``

And in style.xml under `res/values` I changed the statusbar color, I added the theme like so:
```javascript
<resources>
<style name="AccountKitTheme" parent="Theme.AccountKit">
    <item name="com_accountkit_primary_text_color">@android:color/white</item>
    <item name="com_accountkit_status_bar_color">#888888</item>
    <item name="com_accountkit_button_background_color">#71b7e6</item>    
</style>
</resources>
```
It might be a good idea to bake this into the plugin options (at least the status bar color) so that the experience is a bit more seamless.

Welcome back! :-)
